pkgs/top-level/all-packages.nix

```diff
@@ -26041,12 +26041,8 @@ with pkgs;
       overrideCC stdenv buildPackages.llvmPackages.clangNoLibcxx
     else stdenv;
   };
-  busybox-sandbox-shell = callPackage ../os-specific/linux/busybox/sandbox-shell.nix {
-    # musl roadmap has RISC-V support projected for 1.1.20
-    busybox = if !stdenv.hostPlatform.isRiscV && !stdenv.hostPlatform.isLoongArch64 && stdenv.hostPlatform.libc != "bionic"
-              then pkgsStatic.busybox
-              else busybox;
-  };
+  busybox-static = busybox.override { enableStatic = true; };
+  busybox-sandbox-shell = callPackage ../os-specific/linux/busybox/sandbox-shell.nix { };
```

Comment:
Busybox is used by `nix` inside its sandbox for `/bin/sh`, which needs to be self-standing. So I'm pretty sure this *needs* to be built with `useMusl` otherwise it leaves references to other `/nix/store` paths.

Reply:
I don't think this is true.  Just because libc can access some paths, it doesn't mean that the shell will want to, and in fact I'd say it's extremely unlikely.

Reply:
You might be right, it's possible failing to find those references (e.g. glibc locales) is not fatal, but it still feels wrong to have broken `/nix/store` references in the nix sandbox. I'd leave it to nix maintainers to decide.